glyph-brush-layout/src/words.rs

```diff
@@ -117,25 +118,18 @@ where
 
             if !control {
                 let positioned = RelativePositionedGlyph {
-                    relative: caret,
+                    relative: point(caret, 0.0),
                     glyph,
                 };
 
-                if let Some(glyph_bounds) = positioned.bounds() {
-                    if let Some(mut word) = bounds.take() {
-                        word.min.x = word.min.x.min(glyph_bounds.min.x);
-                        word.min.y = word.min.y.min(glyph_bounds.min.y);
-                        word.max.x = word.max.x.max(glyph_bounds.max.x);
-                        word.max.y = word.max.y.max(glyph_bounds.max.y);
-                        bounds = Some(word);
-                    } else {
-                        bounds = Some(glyph_bounds);
-                    }
+                caret += advance_width;
 
+                if positioned.bounds().is_some() {
                     glyphs.push((positioned, color, font_id));
-                }
 
-                caret.x += advance_width;
+                    // not an invisible trail
+                    caret_no_trail = caret;
+                }
```

Comment:
Nice. This should be faster too, right?

Reply:
Not much difference in the benchmarks, I assume the bounds are pre-computed anyway? But simpler & less buggy logic here is always welcome!